### PR TITLE
feat(plugins/backend/okta): hide oauth creds and token from logs and config api

### PR DESCRIPTION
**Pull request by secustor**
Add visibility tags to `token` and `oauth` config option so these get hidden in the config api and from logs. 
The former makes this tokens hidden on the devtools config page. 

#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [x] Added changeset (run `yarn changeset` in the root)
- [ ] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

**Review by punkle — DISMISSED** [review]

**Comment by secustor** [comment]
Still an issue in 0.9.11

**Comment by secustor** [comment]
@punkle Can you reopen this PR, this is still needed to mask credentials.

File: plugins/backend/catalog-backend-module-okta/config.d.ts [hunk 1]
```diff
@@ -22,7 +22,23 @@ export interface Config {
       /**
        * Okta configuration
        */
-      okta?: AccountConfig[];
```
Comment (Xantier) [inline plugins/backend/catalog-backend-module-okta/config.d.ts:25]: This import is now unused right?

Comment (secustor) [reply plugins/backend/catalog-backend-module-okta/config.d.ts:25]: Seems like this introduced somewhere from main. I will remove it.

**Review by Xantier — APPROVED** [review]

**Pull request merged**